contributors/devel/development.md

```diff
@@ -279,29 +279,59 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Install etcd
+
+```sh
+hack/install-etcd.sh
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh scripts
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
```

Comment:
```sh
# Run all the presubmission verification. Then run a specific update script (hack/update-*.sh)
# for each failed verification. For example:
#   hack/update-gofmt.sh (to make sure all files are correctly formatted, usually needed when you add new files)
#   hack/update-bazel.sh (to update bazel build related files, usually needed when you add or remove imports)
make verify 

# Alternatively, run all update scripts to avoid fixing verification failures one by one.
make update
```